composer.json

```diff
@@ -22,14 +22,14 @@
     "email": "info@signify.co.nz"
   },
   "require": {
-    "silverstripe/cms": "^4",
-    "dnadesign/silverstripe-elemental": "^4",
-    "symbiote/silverstripe-gridfieldextensions": "^3",
+    "silverstripe/cms": "^5",
```

Comment:
Could you please change the constraints to `^4 || ^5`; there's no need to shut out Silverstripe 4 sites if there are no problems with compatibility.

Reply:
Thanks @sig-steve , good call.